service-base/src/main/java/org/eclipse/hono/service/auth/device/DeviceCredentials.java

```diff
@@ -39,4 +42,17 @@
      * @return The tenant.
      */
     String getTenantId();
+
+    /**
+     * Gets additional properties that can be used to get credentials from the Credentials API.
```

Comment:
IMHO this should describe the purpose of the method in a more abstract way, not by means of one of its potential use cases.